source/braille.py

```diff
@@ -2870,7 +2873,7 @@ def terminate(self):
 		super().terminate()
 		# Clear the display.
 		try:
-			self.display([0] * self.numCells)
+			handler.buffer.clear()

```

Comment:
Shoudln't this be
```suggestion
			self.buffer.clear()
```

Reply:
I don't think so - I don't believe `BrailleDisplayDriver` has a `buffer` attribute

Reply:
Ah i'm sorry, I read the diff incorrectly and thought that this was a function on the handler.